src/main/java/devcourse/jpa/domain/order/Order.java

```diff
@@ -0,0 +1,69 @@
+package devcourse.jpa.domain.order;
+
+import devcourse.jpa.domain.member.Member;
+import devcourse.jpa.domain.orderitem.OrderItem;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "order_id")
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "member_id")
+    private Member member;
+
+    @Enumerated(value = EnumType.STRING)
+    private OrderStatus orderStatus;
+
+    private LocalDateTime orderDateTime;
+
+    @Lob
+    private String memo;
+
+    @OneToMany(mappedBy = "order")
+    private List<OrderItem> orderItems = new ArrayList<>();
+
+    public Order() {
+    }
+
+    public Order(Member member) {
+        this.member = member;
+    }
+
+    public void addOrderItem(OrderItem orderItem) {
+        this.orderItems.add(orderItem);
+    }
```

Comment:
```suggestion
    public void addOrderItem(OrderItem orderItem) {
        this.orderItems.add(orderItem);
        orderItem.setOrder(this);
    }
```
영속전에는 OrderItem에 Order가 null이라서, 값을 넣어놓는게 좋을 것 같아요. (실수 방지 NPE 발생가능성)

또 `this.orderItems.add(orderItem); `이 부분도 동일한 orderItem을 넣었을 때, 어떻게 처리할지 고민해보시면 좋을 것 같아요. (중복으로 저장할지? 예외를 던질지? 등등..)